rllib/tests/run_regression_tests.py

```diff
@@ -39,7 +39,7 @@
     required=True,
     help="The directory in which to find all yamls to test.",
 )
-parser.add_argument("--num-cpus", type=int, default=8)
+parser.add_argument("--num-cpus", type=int, default=None)
```

Comment:
A machine during test may not have 8 cpus.

Reply:
maybe leave this change out for now as well?
is None the default value for num_cpus parameter for ray.init()?

Reply:
yes, `None` is the default. That's why `None` should also be the default here. otherwise you can't run default regression on machine with less cpus.